{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 5",
      "fix_state": "Not affected",
      "package_name": "conga",
      "cpe": "cpe:/o:redhat:enterprise_linux:5"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2012-11-06T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2012-5493 conga (Plone): Restricted Python sandbox escape",
    "id": "874704",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=874704"
  },
  "cvss": {
    "cvss_base_score": "6.0",
    "cvss_scoring_vector": "AV:N/AC:M/Au:S/C:P/I:P/A:P",
    "status": "draft"
  },
  "cvss3": {
    "cvss3_base_score": "",
    "cvss3_scoring_vector": "",
    "status": ""
  },
  "iava": "",
  "cwe": "",
  "statement": "Not vulnerable. This issue did not affect the versions of luci (as provided by conga) as shipped with Red Hat Enterprise Linux 5.",
  "acknowledgement": "",
  "name": "CVE-2012-5493",
  "document_distribution": "",
  "details": [
    "gtbn.py in Plone before 4.2.3 and 4.3 before beta 1 allows remote authenticated users with certain permissions to bypass the Python sandbox and execute arbitrary Python code via unspecified vectors."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2012-5493\nhttps://nvd.nist.gov/vuln/detail/CVE-2012-5493"
  ]
}